### DISCUSSION TITLE
AdventCalendarの作成者にカレンダーの表紙のViewsが見えると嬉しいかも。

### DISCUSSION THREAD

**Discussion by kaizen-nagoya** [increments/qiita-discussions#155] [要望・アイデア・不具合]
AdventCalendarの作成者にカレンダーの表紙のViewsが見えると嬉しいかもしれないと思い記載します。

複数のAdventCalendarを作成してみると、参加予定者数、記事数などから、次の手を打つことをかんがえようとしています。カレンダーの表紙のViewsがわかると、カレンダーそのものへの興味の度合いから、何に手を打つといいかが予測しやすくなります。

Viewsが多いのに、投稿予定者、講読予定者が少ないのであれば、考え方をより深くか、より広く示すような漢字。

Viewsが少ないのであれば、まず存在そのものを知ってもらうようにするとか。

自分が投稿したカレンダー記事のviewsとの比較をすると、より構造的な戦略が立てられるかもしれません。

**Comment by kskwtnk** [comment]
@kaizen-nagoya 
ご意見ありがとうございます！
カレンダーを作成した後、参加者を募る動きも実施していたkaizen-nagoyaさんの意見だとリアルさがあってありがたいです 🙏 
（参加者を募集している様子をたまたまTwitterでお見かけしました）

こちら、次回のAdvent Calendar開発時の検討リストに入れておきます！
